css/tools.less

```diff
@@ -69,15 +68,41 @@
     }
 
     a           span:before { content: '\f101'; }
+    a.recent    span:before { content: '\f307'; }
+    a.index     span:before { content: '\f507'; }
+    a.media     span:before { content: '\f473'; }
+}
 
+.user-tools {
+    a span {
+        display: inline-block;
+        width: 20px;
+        height: 20px;
+        -webkit-font-smoothing: antialiased;
+        font-size: 20px;
+        line-height: 20px;
+        vertical-align: middle;
+        font-family: 'Genericons';
+        text-decoration: inherit;
+        font-weight: normal;
+        font-style: normal;
+    }
+
+    a           span:before { content: '\f101'; }
     a.admin     span:before { content: '\f445'; }
     a.login     span:before { content: '\f427'; }
     a.logout    span:before { content: '\f470'; }
     a.register  span:before { content: '\f425'; }
     a.profile   span:before { content: '\f304'; }
-    a.recent    span:before { content: '\f307'; }
-    a.index     span:before { content: '\f507'; }
-    a.media     span:before { content: '\f473'; }
+
+    p.user:before {
+        font-family: 'bootstrap-icons';
```

Comment:
Minor: You are using Bootstrap Icons before you include the necessary files for it. As they are in the very next commit, this is more a technicality and not that important. But generally you'd always want a commit to be able to stand on its own.

Reply:
Yeah I think it's timing. I used `git add --patch` for the first and went using the first prompt. Then added anything related to that prompt. Comit and repeat. I will try to avoid this when using patch